src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/BindingMetadataProviderContext.cs

```diff
@@ -36,6 +39,16 @@ public class BindingMetadataProviderContext
         public ModelMetadataIdentity Key { get; }
 
         /// <summary>
+        /// Gets the property attributes.
+        /// </summary>
+        public IReadOnlyList<object> PropertyAttributes { get; }
```

Comment:
This is on a class that's created and then immediately thrown away so I'm not concerned about the extra two fields. Let me know if you think this is important.